src/java_bytecode/java_utils.cpp

```diff
@@ -20,3 +22,94 @@ bool java_is_array_type(const typet &type)
     to_struct_type(type).get_tag()),
                     "java::array[");
 }
+
+unsigned java_parameter_count(
```

Comment:
Duplicate of https://github.com/diffblue/cbmc/blob/test-gen-support/src/java_bytecode/java_types.cpp#L162 which was used to generate the method's `code_typet` to begin with. How about making https://github.com/diffblue/cbmc/blob/test-gen-support/src/java_bytecode/java_bytecode_convert_method.cpp#L111 public and then iterating over the `code_typet`'s formal parameters?